Small Tweaks for OSI Metric (#99)
chaoss/wg-risk: metrics/OSI_Approved_Licenses.md

## Objectives

> Identify whether a project has licenses present which do not conform to the OSI definition of open source. This transparency helps projects make conscious decisions about whether or not to include licenses that are not approved by OSI.

Reviewer: okay

Extract list of licenses from a code base, same as in License Coverage metric.
Compare the list of licenses to [Licenses.json](https://raw.githubusercontent.com/spdx/license-list-data/master/json/licenses.json) and take note of how many licenses are approved by the OSI.

> Calculate the percentage of OSI approved files on the SPDX license list.

Reviewer: This doesn't make sense nor did the orginal sentence make sense.

Maybe this is better?
```suggestion
Calculate the percentage of files with an on the OSI approved license.
```
